.github/workflows/dockerhub.yml

```diff
@@ -1,12 +1,11 @@
-
-
 name: Docker Hub build
-
+# This workflow will build and push of docker images when new push occurs to develop, master, release branches but doesn't run any unit test or sonar scanner
```

Comment:
Minor typo: the 'of' can be removed, or maybe it was meant to be 'official'?

Reply:
updated